src/monarch_ingest/utils/export_utils.py

```diff
@@ -140,6 +141,8 @@ def generate_tsv(tsv_fh, solr, filters):
     for filter in filters:
         if filter == 'category:"biolink:DiseaseToPhenotypicFeatureAssociation"':
             golr_params['fl'] += ',' + disease_to_phenotype_extra_fields
+        elif 'GeneToGene' in filter:
+            golr_params['fl'] += ',' + gene_to_gene_extra_fields
```

Comment:
you could remove line 129 and just change this to 
`golr_params['fl'] += ',' + ','.join(['object_taxon', 'object_taxon_label'])`
otherwise looks good